tic_tac_toe/game.py

```diff
@@ -24,17 +26,28 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if not isinstance(position, tuple):
+        return False
+    if len(position) != 2:
+        return False
+    if position[0] not in [0,1,2]:
```

Comment:
Overall reasonable, but might have been cleaner to first split out the parts of position by doing something like `row, col = position` and working with those. Matter of style preference there, I guess.

(take this comment to apply to most places you were doing position[0]/position[1]. Again, a matter of style pref though)